src/include/brain/indextune/lspi/lstd.h

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// lstd.h
+//
+// Identification: src/include/brain/indextune/lspi/lstd.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "brain/util/eigen_util.h"
+
+/**
+ * Least Squares Temporal-Differencing Estimator(LSTD(0))
+ * References:
+ *  [1] Cost Model Oblivious DB Tuning by Basu et. al.
+ *  [2] Linear Least-Squares Algorithms for Temporal Difference Learning by
+ *Barto et. al.(Page 13) The Least Squares TD Estimator(based on the Recursive
+ *least squares formulation) provides an efficient way to evaluate the value
+ *function of a parameterized state.
```

Comment:
I didn't check the equation here. But I think they're using the *Least-squares policy iteration by MG Lagoudakis - ‎2003* instead of *Temporal Difference Learning by Barto et. al*, right? Are you aware of this? Is this a typo?

Reply:
Yep, I'm aware that they've written about using LSPI - But in their own code there is only one `LSTD` class and no mention of `LSTDQ` - That's why I initially looked up LSTD to try and understand what they've done.

Reply:
In any case - I'm guessing it must be a typo on their end - I just checked and the formula matches their paper - I'll fix the reference there.